### PR TITLE
XBTS update locales

### PR DESCRIPTION
**Pull request by technologiespro**
- Update XBTS locales: gateway.assets
- Add CoffeeCoin (COF) icon
- Branding: remove asset XBTSX.KEC

### CODE REVIEW

**Comment by technologiespro** [comment]
- ### 7ce75e4: Fix white screen on exchange page:locale:ru
When trading assets, example OPEN.ETH_BTS, symbol **%** page crash occurs in ru:locale (tooltip.gateway)

- ### 4c3ed04: On page member-stats ru:locale
Layout destruction due to cyrillic invalid translation tokens

File: app/help/ru/components/AccountMembership.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-[# пожизненно]
+[# lifetime]
```
Comment (sschiessl-bcp) [inline app/help/ru/components/AccountMembership.md]: Why did you put english titles here now?

Comment (technologiespro) [reply app/help/ru/components/AccountMembership.md]: reverted ru

Comment (sschiessl-bcp) [reply app/help/ru/components/AccountMembership.md]: There are two more cases like that in the PR

**Pull request merged**